release/models/platform/openconfig-platform.yang

```diff
@@ -472,11 +478,14 @@ module openconfig-platform {
 
     leaf last-switchover-time {
       type oc-types:timeticks64;
+      units "nanoseconds";
```

Comment:
Technically, I think this is now stated in the type (https://github.com/openconfig/public/blob/master/release/models/types/openconfig-types.yang#L106), but I am OK with restating it here.